README.md

```diff
@@ -364,6 +364,28 @@ reached. If no configuration file is found, Regal will use the default configura
 A custom configuration may be also be provided using the `--config-file`/`-c` option for `regal lint`, which when
 provided will be used to override the default configuration.
 
+### Loading Capabilities from URLs
```

Comment:
This section should be under `## Capabilities` line 533 in your PR.

Reply:
b5cb5c5